tracing-subscriber/src/filter/targets.rs

```diff
@@ -277,6 +277,59 @@ impl Targets {
         self
     }
 
+    /// Returns the default level for this filter, if set.
+    ///
```

Comment:
maybe worth adding something like:
```suggestion
    ///
    /// The default level is used to filter any spans or events with targets
    /// that do not match any of the configured set of prefixes.
    ///
```